### PR TITLE
chore: update readme for E2E behavior on WSL

### LINKED ISSUE #853: E2E tests not running on Ubuntu/WSL
## Describe the bug

e2e tests are not being able to run on ubuntu wsl.

**To Reproduce**

1. Clone the latest code
2. run `yarn build`
3. run `yarn test:e2e`

## Expected behavior

e2e test cases should work

## Screenshots

image

## Extension (please complete the following information)

- OS: Windows 10/ WSL (ubuntu)
- Version: Dev

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?
Yes

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

Update `Readme` to have a specific PS note linking to the issue that I created for running E2E on WSL.

#### Pull request checklist

- [x] Addresses an existing issue: Fixes #853 
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ Please ensure that you have at least the **minimum** recommended versions
 
 We use [jest](https://github.com/facebook/jest) as our test framework and [puppeteer](https://github.com/GoogleChrome/puppeteer) for browser automation in our end-to-end UI tests.
 
+> We have a known issue with running E2E on WSL in Windows 10. Issue posted [here](https://github.com/microsoft/accessibility-insights-web/issues/853).
```
Comment (smoralesd) [inline README.md]: should we rephrase this to something like???:
`There is a know issue`
As mention before... this is not necessarily an issue with our code or deps, this is a known limitation of WSL (out of the box, power users may be able to get graphical applications running...)

Comment (dbjorge) [reply README.md]: +1. I don't think "known issue" is a fair description. I think it would be more reasonable to instead say something like "this project's end to end tests require the ability to run a non-headless chromium process. Because of this, they are incompatible with non-graphical development environments (notably, a default WSL environment on Windows)"

Comment (Shobhit1) [reply README.md]: I will take that line @dbjorge . Thanks.
